compute/admin_guide/install/system_requirements.adoc

```diff
@@ -358,9 +365,9 @@ Native Kubernetes 1.22.9 (containerd 1.6.4)
 
 Native Kubernetes 1.23.3 (containerd 1.4.12)
 
-Native Kubernetes - 1.22.9 Docker 20.10.16
+Native Kubernetes 1.23.8 Docker 20.10.17
 
-Native Kubernetes - 1.23.3 Docker 20.10.12
+Lightweight Kubernetes (k3s) 1.21.7+k3s1 (containerd 1.4.12-k3s1)
```

Comment:
I can't find this in the QA list https://confluence.paloaltonetworks.local/display/TWIS/Kepler+system+requirements. Do I miss anything?

Reply:
Yes, it's easy to miss because they have it listed only as k3s. I got clarification from QA and I'm updating the tables to reflect that. Thanks @MorShabiPAN

Reply:
I've updated the table, you can find it now in its proper position in the supported x86 orchestrators.